infer_subc/organelles/membrane.py

```diff
@@ -0,0 +1,279 @@
+import numpy as np
+from skimage.morphology import disk, ball, binary_closing, closing, binary_dilation, dilation, binary_erosion
+from infer_subc.core.img import fill_and_filter_linear_size, get_interior_labels, get_max_label, inverse_log_transform, log_transform, masked_inverted_watershed, masked_object_thresh, select_channel_from_raw, threshold_otsu_log
+from aicssegmentation.core.utils import hole_filling
+from infer_subc.organelles.nuclei import infer_nuclei_fromlabel
+from infer_subc.organelles.cellmask import non_linear_cellmask_transform
+from infer_subc.constants import (
+    TEST_IMG_N,
+    NUC_CH,
+    LYSO_CH,
+    MITO_CH,
+    GOLGI_CH,
+    PEROX_CH,
+    ER_CH,
+    LD_CH,
+    PM_CH,
+    RESIDUAL_CH,
+)
+
+def rescale_intensity(in_img: np.ndarray):
```

Comment:
Commented on this function in cellmask.py with something that may negate the need for this function. If it is needed, let's only keep it in one location. My suggestion for location is in infer_subc.core.img and can then be imported here and the other location as needed.

Reply:
went ahead and removed this from the code and used minmaxintensity function